core-designsystem/src/androidMain/kotlin/io/github/droidkaigi/confsched2022/designsystem/components/KaigiTopAppBar.kt

```diff
@@ -60,3 +62,27 @@ fun KaigiTopAppBar(
         )
     )
 }
+
+@Preview
+@Composable
+private fun KaigiTopAppBarPreview() = KaigiTopAppBar(
+    showNavigationIcon = true,
+    onNavigationIconClick = {},
+    title = {
+        Text(
+            text = "KaigiTopAppBarPreview",
+        )
+    }
+)
+
+@Preview
+@Composable
+private fun KaigiTopAppBarPreview_HiddenNavigationIcon() = KaigiTopAppBar(
```

Comment:
I'm a little concerned about whether I need to have two Previews since I just need to change the showNavigationIcon